### PR TITLE
Create local cars when starting the server

### PR DESCRIPTION
**Pull request by mahdibenromdhane**

### CODE REVIEW

File: src/assets/cars/cars-definition.json [hunk 1]
```diff
@@ -1,25 +1,90 @@
 [
   {
-    "Vehicle_ID": 100000,
-    "Vehicle_Make": "GOUPIL",
-    "Vehicle_Model": "G5 benne",
-    "Vehicle_Model_Version": "",
+  "Vehicle_ID": 100000,
```
Comment (LucasBrazi06) [inline src/assets/cars/cars-definition.json:3]: Tabulation not correct?

File: src/integration/car/local/LocalCarIntegration.ts [hunk 2]
```diff
@@ -0,0 +1,235 @@
+import { CarCatalog, CarCatalogChargeAlternativeTable, CarCatalogChargeOptionTable, CarCatalogConverter } from '../../../types/Car';
```
Comment (LucasBrazi06) [inline src/integration/car/local/LocalCarIntegration.ts:1]: Not the right place, Car integration class must implement the interface which is not the case here.
Move this file to the newly created directory 'bootstrap' where I put the CS Template import.
Proposal for the naming: LocalCarCatalogBootstrap.uploadLocalCarCatalogsFromFile()

**Review by LucasBrazi06 — CHANGES_REQUESTED** [review]
Small changes

File: src/assets/cars/cars-definition.json (hunk 1 above)
Comment (mahdibenromdhane) [reply src/assets/cars/cars-definition.json]: Fixed

File: src/integration/car/local/LocalCarIntegration.ts (hunk 2 above)
Comment (mahdibenromdhane) [reply src/integration/car/local/LocalCarIntegration.ts]: Done

**Review by LucasBrazi06 — APPROVED** [review]

**Pull request merged**